bridges/FacebookBridge.php

```diff
@@ -736,6 +741,7 @@ private function collectUserData(){
 						$item['title'] = htmlspecialchars_decode($title, ENT_QUOTES);
 						$item['author'] = htmlspecialchars_decode($author, ENT_QUOTES);
 						$item['timestamp'] = $date;
+						$item['icon'] = htmlspecialchars_decode($profilePic, ENT_QUOTES);
```

Comment:
It is only returned via Json and Plaintext formats. Maybe you need to use https://github.com/RSS-Bridge/rss-bridge/wiki/The-getIcon-function instead?